src/scaffoldmaker/scaffoldpackage.py

```diff
@@ -271,8 +270,12 @@ def generate(self, region, applyTransformation=True):
             # define user AnnotationGroups from serialised Dict
             self._userAnnotationGroups = [ AnnotationGroup.fromDict(dct, self._region) for dct in self._userAnnotationGroupsDict ]
             self._isGenerated = True
-            if applyTransformation:
-                self.applyTransformation()
+            fieldmodule = self._region.getFieldmodule()
+            for editFieldName in ['coordinates', 'inner coordinates']:
+                editCoordinates = fieldmodule.findFieldByName(editFieldName)
+                if editCoordinates.isValid():
+                    if applyTransformation:
```

Comment:
`if applyTransformation:` should be the first statement above `fieldmodule = ...` as no need to recheck for every field.
This application to coordinates and inner coordinates is what ScaffoldCreator needs to do on clicking Done as well.